src/main.js

```diff
@@ -887,6 +924,16 @@ class MiniGraphCard extends LitElement {
     return this._hass.callApi('GET', url);
   }
 
+
+  _convertState(res) {
```

Comment:
Again used this "res" name trick to make eslint happy but actually in this case I could just return the item/res and use assignment and map function above - where I call _convertState. Up to you - please let me know :)

Reply:
I don't mind either really, as long as eslint is happy lol, and we do it on a few other placed already so I wouldn't worry about it. 😉